<?xml version="1.0"?>
<launch>

    <!-- Launches the skills simulation action servers -->
    <!-- <node name="driveSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="driveSkill $(arg robotId)"></node> -->
    <!-- <node name="driveEdgesSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="driveEdgesSkill $(arg robotId)"></node> -->
    <!-- <node name="locateSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="locateSkill $(arg robotId)"></node> -->
    <!-- <node name="pickSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="pickSkill $(arg robotId)"></node> -->
    <!-- <node name="placeSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="placeSkill $(arg robotId)"></node> -->
    <!-- <node name="waitSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="waitSkill $(arg robotId)"></node> -->
    <!-- <node name="dockSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="dockSkill $(arg robotId)"></node> -->
    <!-- <node name="actuateArmSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="actuateArmSkill $(arg robotId)"></node> -->
    <node name="MoveArmSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="MoveArmSkill"></node>
    <node name="GenericSkill" pkg="skill_simulation_server" type="skill_action_server.py" output="screen" args="GenericSkill"></node>

</launch>
